### PR TITLE
doc: proposal template & instructions

### PR DESCRIPTION
**Pull request by tbruyelle**
This PR contains a suggestion for a proposal template that can be used to create new proposal in the repo.
Also adds instructions in the README.

### CODE REVIEW

File: XXX_proposal_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Status
+> The proposal lifecycle is 
+> - Draft when not on chain and open for discussions
+> - Abandonned when not on chain and will never be
+> - Deposit when on chain in the deposit period
+> - Voting when on chain in the voting period
+> There's no need for Accepted or Rejected status as the govgen proposals have
+> a voting period of 365 days.
+
+[Draft|Abandonned|Deposit|Voting]
```
Comment (tbruyelle) [inline XXX_proposal_template.md:15]: Maybe should be just have `[Draft|Abandonned|On Chain]`, so we don't have to update it once it's on chain, we can just forget it.

**Comment by tbruyelle** [comment]
> @tbruyelle Should we add different types of proposals they can submit or not needed? I suppose this could be added to the proposal ID? (E.g. signaling, parameter change, community, software upgrade, etc.)

Good point, the template aims only text proposals for AtomOne, I'm going to add this information.

File: XXX_proposal_template.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+<!--
+This template is designed to draft a text proposal for AtomOne. This draft will
+be used to start discussions and potentially submit it on the GovGen chain. 
+
+It is a blend between the proposal content we can see in the Cosmos forums, and
+the more technical ADR (Architecture Decision Records). The intent of such a
+mix is to ensure that proposers go into all the details of their proposals.
+
+A GovGen text proposal has a voting period of 365 days, requires a min deposit
+of 50govgen and a deposit of 5,000govgen.
```
Comment (giunatale) [inline XXX_proposal_template.md:12]:
```suggestion
of 50 $GOVGEN that will need to come from the initial onchain proposer, 
and a total deposit of 5,000 $GOVGEN that can be permissionlessly funded
by by any account with balance with arbitrary amounts.
```

**Review by giunatale — APPROVED** [review]

**Review by giunatale — CHANGES_REQUESTED** [review]
sorry, previously approved by mistake, but thereis only a minor change request, the rest looks good TY!

File: XXX_proposal_template.md (hunk 2 above)
Comment (tbruyelle) [reply XXX_proposal_template.md:12]: Can you update your suggestion plz? there's `by` repeated twice in the last line.

Comment (giunatale) [reply XXX_proposal_template.md:12]: done!

**Review by giunatale — APPROVED** [review]

**Comment by kristovatlas** [comment]
LGTM

**Pull request merged**